modules/@apostrophecms/ui/ui/apos/components/AposContextMenuDialog.vue

```diff
@@ -77,11 +77,11 @@ export default {
 
 <style lang="scss">
 
-.apos-context-menu__dialog--unpadded .apos-context-menu__pane  {
+.apos-context-menu__dialog--unpadded > .apos-context-menu__pane {
```

Comment:
dialogs are now inside one another, modifiers should only modify their local elements